VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,254 @@
+# Validators FAQ
+
+## General concepts
+
+### What is a validator?
+
+The Cosmos Hub is based on Tendermint, which relies on a set of validators to secure the network. The role of validators is to run a full-node and participate in consensus by broadcasting votes which contain cryptographic signatures signed by their private key. Validators commit new blocks in the blockchain and receive rewards in exchange for their work. They must also participate in governance by voting on proposals.
```

Comment:
Maybe finish with `Validators are weighted according to how much they have staked`. Good segue into "What is staking"

Reply:
Ok